lib/generators/manageiq/plugin/templates/bin/rails

```diff
@@ -4,6 +4,8 @@
 
 ENGINE_ROOT = File.expand_path('../..', __FILE__)
 ENGINE_PATH = File.expand_path('../../lib/<%= plugin_path %>/engine', __FILE__)
+APP_PATH    = File.expand_path('../../spec/manageiq/config/application', __FILE__)
```

Comment:
@Fryguy This is a blaster outer overflow right?  You can't have spec/manageiq in the main manageiq repo?  Manageiq inception...

Reply:
oh wait, this is the generator